src/components/layout/footer/index.tsx

```diff
@@ -9,7 +9,7 @@ const Footer = () => {
   return (
     <footer
       className={clsxm(
-        'mt-10 flex w-full flex-col items-center border-t border-t-gray-200 py-5 pr-8 text-sm text-gray-500 transition-all duration-500 md:flex-row md:text-base',
+        'mt-auto flex w-full flex-col items-center border-t border-t-gray-200 py-5 pr-8 text-sm text-gray-500 transition-all duration-500 md:flex-row md:text-base',
```

Comment:
I think is better to use `mt-10` instead `mt-auto` cause we need to add space between footer and content